README.md

```diff
@@ -252,7 +252,7 @@ ActiveAdmin.register Page do
     actions
   end
 
-  form do |f|
+  form html: { class: "edit_atomic_content" } do |f|
```

Comment:
Yeah. I really don't like activeadmin / formtastic for this stuff. Its gonna be a huge limiter in the very near future.